tests/sentry/api/endpoints/test_organization_details.py

```diff
@@ -227,6 +211,52 @@ def test_various_options(self):
         assert options.get('sentry:sensitive_fields') == ['password']
         assert options.get('sentry:safe_fields') == ['email']
 
+    def test_setting_legacy_rate_limits(self):
+        org = self.create_organization(owner=self.user)
+        self.login_as(user=self.user)
+        url = reverse(
+            'sentry-api-0-organization-details', kwargs={
+                'organization_slug': org.slug,
+            }
+        )
+        response = self.client.put(
+            url,
+            data={
+                'accountRateLimit': 1000,
+            }
+        )
+        assert response.status_code == 400, response.content
```

Comment:
Why does this return a 400?

Reply:
you cannot configure this option if you haven't done so previously, so attempting to set it returns an error